app/views/content/steps-to-become-a-teacher/_step_4_find_a_teacher_training_course.html.erb

```diff
@@ -6,6 +6,7 @@
 
 <p>If you already have a degree, find out <a href="/train-to-be-a-teacher/how-to-choose-your-teacher-training-course">how to choose your postgraduate teacher training course</a>.</p>
 
+<p>If you're not sure which route into teaching is right for you, you can <a href="/routes-into-teaching">explore the courses available based on your circumstances</a>.</p> 
```

Comment:
```suggestion
<p>If you're not sure which route into teaching is right for you, you can <a href="/routes-into-teaching">answer 3 questions and find routes into teaching based on your circumstances.</a>.</p> 
```

I think `explore the courses` might be a bit confusing given that we've just talked about postgraduate and undergraduate **courses** - we are mixing up routes and courses? But reword my suggestion - it's a bit repetitive!